handel/src/services/apigateway/swagger/apigateway-swagger-template.yml

```diff
@@ -92,6 +92,36 @@ Resources:
         Value: {{this}}
       {{/each}}
       {{/if}}
+
+  {{#if warmup}}
+  LambdaWarmupPermission{{@index}}:
+    Type: "AWS::Lambda::Permission"
+    Properties:
+      Action: 'lambda:InvokeFunction'
+      FunctionName: !Ref LambdaFunction{{@index}}
+      Principal: events.amazonaws.com
```

Comment:
Like in the other template, it would be great if we could put a resource constraint here if possible.